pkg/addons/velero/velero.go

```diff
@@ -135,7 +135,11 @@ func (o *Velero) Outro(ctx context.Context, cli client.Client) error {
 	return nil
 }
 
+type Options struct {
+	IsEnabled bool
+}
```

Comment:
why the change to pass an options struct? personally I want to have to fix the compile errors for each call to 'new' when I add a new field, as it's almost always going to be required

Reply:
restored the original arguments.